snippets/core/system-text-json/csharp/IMyInterface.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace SystemTextJsonSamples
+{
+    public interface IMyInterface
```

Comment:
This sample has "placeholder" names for the types. It would be better to have a more scenario driven name (similar to WeatherForecast) that the reader can more easily understand. I can try to come up with something if you like, but using some canonical sample example that gets used to explain folks about interfaces would work better.